packages/front-b2b-link-rn/src/Front.tsx

```diff
@@ -53,9 +52,12 @@ const FrontFinance = (props: {
     ) {
       setShowWebView(false);
     }
+    if (type === "showClose") {
+      props.onClose && props?.onClose();
+    }
     if (type === "brokerageAccountAccessToken") {
       setPayload(payload);
-      props.onReceive && props.onReceive(payload);
+      props.onReceive && props?.onReceive(payload);
```

Comment:
why do we need `props?` here?

Reply:
firstly i thought we need to fetch the tokens and pass into another api from client app so if it is not required then we just simply remove it. BTW this payload has any purpose, i mean do we need this to fetch payload because there is no use if we remove the onReceive method